src/main/kotlin/commands/SlowCommand.kt

```diff
@@ -0,0 +1,46 @@
+package commands
+
+import Command
+import PermissionTypes
+import Permissions
+import StringHelper
+import StringHelper.MessageTypes.MISSING_PERMISSIONS
+import arg
+import doesLater
+import integer
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+
+object SlowCommand : Command("slow") {
+    init {
+        integer("wait") {
+            integer("time") {
+                doesLater { context ->
+                    if (!Permissions.hasPermission(message, PermissionTypes.COUNCIL_MEMBER)) {
+                        StringHelper.sendMessage(message.channel, MISSING_PERMISSIONS)
+                        return@doesLater
+                    }
+                    val wait: Int = context arg "wait"
+                    val time: Int = context arg "time"
+                    val originalWait = message.serverChannel!!.rateLimitPerUser
+                    message.serverChannel!!.edit {
+                        rateLimitPerUser = wait
+                    }
+                    GlobalScope.launch {
+                        delay(time * 1000L)
+                        message.serverChannel!!.edit {
+                            rateLimitPerUser = originalWait
+                        }
+                    }
```

Comment:
what is this for???

Reply:
removes slow mode from the channel after the time given in the command, this wont work if the bot restarts though

Reply:
Oh yea. Can you remove this? You'll have to wait for me to make the mute / slowmode system

Reply:
Yeah sure. Should I remove the second argument as well as it is useless without that block?

Reply:
Yea